batch_processing.sh

```diff
@@ -58,7 +58,7 @@ start=$(date +%s)
 if [[ "$SCT_BP_DOWNLOAD" == "1" ]]; then
   sct_download_data -d sct_example_data
 fi
-cd data/sct_example_data
+cd "$SCT_DIR/data/sct_example_data"
```

Comment:
Could we also get rid of these two lines near the bottom?
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/4a5e9edb25501c11ca76d55cdbb9156bba94b51f/batch_processing.sh#L258-L259

Since the script now does `cd` to an absolute path, there's no meaningful sense in which this is going "back".

(Also, normally scripts get executed in a subshell, which means that any `cd` in the script doesn't affect the user's current path after the script exits. (An exception would be if the script is run via the shell builtin command `source`.))

Reply:
Oh, oops! My Ctrl+F missed an instance of `../../..`. Thanks for catching this! :hearts:

Reply:
Ah! It looks like removing the `cd ../../..` revealed another spot where relative paths were being used: 

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/8a40bc28c5ba61d2de8991c0eadc398a9aa7c220/batch_processing.sh#L270-L271

I'll make sure to remove the `.` and replace them with `$SCT_DIR`

Reply:
Fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4150/commits/4ab2471931e01c66530471171f255190050e919a. :)